features/integration-github-bitbucket-gitlab.md

```diff
@@ -14,3 +14,60 @@ You can then either create a new organization directly synchronized with a GitHu
 
 **Update web hooks** notifying Packagist of new code pushes will be installed in the background, and **credentials to access the package** will be **configured automatically**.
+
+## Integrations
+
+Private Packagist integrates with the following systems:
+
+#### GitHub
+* OAuth: Users authenticate on Private Packagist with their GitHub accounts
+* Synchronization:
+    * Keeps team members and access permissions in sync with your GitHub organisations
+    * Synced repositories use the default repository permission on GitHub
+* Credentials: GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
+
+#### GitHub Enterprise
+* OAuth: Users authenticate on Private Packagist with their Bitbucket accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/github-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps team members and access permissions in sync with your GitHub Enterprise organisations
+* Credentials: GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
+
+#### Bitbucket.org / Cloud
+* OAuth: Users authenticate on Private Packagist with their Bitbucket.org accounts
+* Synchronization:
+    * Keeps users permissions in sync with your Bitbucket groups
+    * Group and user changes aren't kept in sync
+* Credentials: Bitbucket API Key or Bitbucket App Password
+* Webhooks: Code changes
+
+#### Bitbucket / Stash Server
+* OAuth: Users authenticate on Private Packagist with their Bitbucket accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/bitbucket-server-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps users and access permissions in sync with your Bitbucket projects
+    * Individual collaborators aren't supported
+    * New projects or team changes aren't kept in sync
+* Credentials: SSH keys
+* Webhooks: Code changes
+
+#### GitLab
+* OAuth: Users authenticate on Private Packagist with their GitLab accounts
+* Synchronization:
+    * Keeps access permissions in sync with your Gitlab groups
+    * Individual collaborators aren't kept in sync
+    * New projects or team changes aren't kept in sync
+* Credentials: GitLab API token
+* Webhooks: Code changes
+
+#### GitLab (Self-hosted)
+* OAuth: Users authenticate on Private Packagist with their Bitbucket accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/gitlab-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps access permissions in sync with your Gitlab groups
+    * Individual collaborators aren't kept in sync
```

Comment:
`Individual collaborators` are not yet supported. Not kept in sync implies that they are added